docs/content/3.components/context-menu.md

```diff
@@ -212,6 +213,16 @@ name: 'context-menu-custom-slot-example'
 You can also use the `#item`, `#item-leading`, `#item-label` and `#item-trailing` slots to customize all items.
 ::
 
+### With color items
```

Comment:
Could you move this under checkbox items?

Reply:
Moved it, now it's under checkbox.